### DISCUSSION TITLE
FIGMA - NEED HELP - PLEASE: Inserting scrollable accordion list into frame with icons and toggle switches

### DISCUSSION THREAD

**Discussion by LeFinneus** [community/community#143469] [Programming Help]
### Select Topic Area

Question

### Body

I am new to Figma and am trying to figure out the software as I create a mobile app prototype.

But now I’m stuck.

I am attempting to create an accordion list with corresponding icons on the left of the headings and a toggle switch to the right of each heading—but left of the up arrow (open button).

Once that has been accomplished, I need to insert this masterpiece into my user settings screen frame and make it all work smoothly.

What is stumping me is the auto layout feature. I’ve been trying for two days—trial and error—to customize the accordion list (attribution: Pixel Happy) and have the list working, but I cannot figure out how to insert the icons and toggle switches without messing up the layout/functionality.

I am on a time crunch - with only 48 hours to finish this up.

Can anyone please help?

**The working file is located here:** [https://drive.google.com/file/d/1quxnJ6p_vpPAeis16Rrq4lmLBca7aK4o/view?usp=sharing](url)

I really appreciate any help you can provide.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @LeFinneus , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it for you. Good luck!
